aixtend/modules/model.py

```diff
@@ -189,4 +194,84 @@ def run_async(self, data: str, name: str = "model_process"):
             response = { 'status': 'FAILED' }
             if resp is not None:
                 response['error'] = resp
-        return response
\ No newline at end of file
+        return response
+    
+    def __get_model_info(self, model_info_json):
+        """Coverts Json to ModelInfo object
+
+        Args:
+            model_info_json (dict): Json from API
+
+        Returns:
+            ModelInfo: Coverted ModelInfo object
+        """
+        m_info = ModelInfo(model_info_json['name'], model_info_json['id'], model_info_json['supplier']['id'])
+        return m_info
+
+    def get_models_from_page(self, page_number: int, task: str, input_language: str = None, output_language: str = None) -> List:
```

Comment:
I would prefer if we make a ModelFilter configuration and pass that to this class instead as we progress in the future. Don't address this now.